pydantic/schema.py

```diff
@@ -807,6 +839,11 @@ def go(type_: Any) -> Type[Any]:
     return ans
 
 
+def normalize_model_name(name: str) -> str:
```

Comment:
```suggestion
def normalize_name(name: str) -> str:
```
? since it applies to both models and enums.

Reply:
Fixed!